resources/developer-mode.mjs

```diff
@@ -147,6 +145,37 @@ export function createUIForSuites() {
     };
     buttons.appendChild(button);
 
+    let i = 0;
+    const kTagsPerLine = 3;
+    for (const tag of Tags) {
+        if (tag === "all")
+            continue;
+        if (i % kTagsPerLine === 0) {
```

Comment:
I think WebKit's coding style is to use !(i % kTagsPerLine) in these cases:
https://webkit.org/code-style-guidelines/#null-false-and-zero